storage/boltz/query_bolt_cursors.go

```diff
@@ -72,9 +71,8 @@ type ReverseBoltCursor struct {
 	BaseBoltCursor
 }
 
-func (f *ReverseBoltCursor) Next() error {
+func (f *ReverseBoltCursor) Next() {
 	f.key, _ = f.cursor.Prev()
```

Comment:
is this correct? Next() is actually Prev()?

Reply:
oops, ReverseCursor

Reply:
:+1: